Installing.py

```diff
@@ -0,0 +1,30 @@
+from PyQt5 import QtCore, QtGui, QtWidgets
+from PyQt5.QtWidgets import *
+from PyQt5.QtGui import QFont, QIcon, QPalette, QColor, QMovie
+import sys
+import os
+
+class Ui_MainWindow(QWidget):
+    def setupUi(self,MainWindow):
+        super().__init__()
+        MainWindow.setObjectName("MainWindow")
+        MainWindow.resize(400,200)
+        self.centralwidget = QtWidgets.QWidget(MainWindow)
+        self.centralwidget.setObjectName("centralwidget")
+        self.label = QtWidgets.QLabel(self.centralwidget)
+        self.label.setGeometry(QtCore.QRect(0,0,400,200))
+        self.label.setMinimumSize(QtCore.QSize(400,200))
+        self.label.setMaximumSize(QtCore.QSize(400,200))
+        self.label.setObjectName("label")
+        MainWindow.setCentralWidget(self.centralwidget)
+        self.movie = QMovie(os.path.join("data","Installing.gif"))
+        self.label.setMovie(self.movie)
+        self.movie.start()
+
+if __name__ == "__main__":
+    app = QtWidgets.QApplication(sys.argv)
+    window = QtWidgets.QMainWindow()
+    ui = Ui_MainWindow()
+    ui.setupUi(window)
+    window.show()
+    sys.exit(app.exec_())
```

Comment:
You can copy and paste this corrected code, but keep in mind what I have changed:
- removed unnecessary imports
- renamed variables to snake_case and the class to CamelCase
- moved the initialization logic (object variables) to the `__init__` method, which then calls the `setup_ui` method to finalize the initialization (reducing the number of calls in the main from 2 to 1)
- fixed the spacing following PEPs (PyCharm actually automatically refactors your spacing if you tell it to)
```suggestion
from PyQt5 import QtCore, QtWidgets
from PyQt5.QtWidgets import *
from PyQt5.QtGui import QMovie
import sys
import os


class UIMainWindow(QWidget):
    def __init__(self, main_window, *args, **kwargs):
        super().__init__()
        self.main_window = main_window
        self.central_widget = QtWidgets.QWidget(self.main_window)
        self.label = QtWidgets.QLabel(self.central_widget)
        self.movie = QMovie(os.path.join("data", "Installing.gif"))
        self.setup_ui()

    def setup_ui(self):
        self.main_window.setObjectName("main_window")
        self.main_window.resize(400, 200)
        self.central_widget.setObjectName("central_widget")
        self.label.setGeometry(QtCore.QRect(0, 0, 400, 200))
        self.label.setMinimumSize(QtCore.QSize(400, 200))
        self.label.setMaximumSize(QtCore.QSize(400, 200))
        self.label.setObjectName("label")
        self.main_window.setCentralWidget(self.central_widget)
        self.label.setMovie(self.movie)
        self.movie.start()


if __name__ == "__main__":
    app = QtWidgets.QApplication(sys.argv)
    window = QtWidgets.QMainWindow()
    ui = UIMainWindow(window)
    window.show()
    sys.exit(app.exec_())

```